sdk/script/management-sdk.ts

```diff
@@ -8,6 +8,10 @@ import Promise = Q.Promise;
 
 import { AccessKey, Account, App, CodePushError, CollaboratorMap, CollaboratorProperties, Deployment, DeploymentMetrics, Headers, Package, PackageInfo, UpdateMetrics } from "./types";
 
+var superproxy = require("superagent-proxy");
+superproxy(superagent);
+var proxy = process.env.http_proxy;
```

Comment:
Thanks so much for contributing this!! Could we make this environment variable upper-case (`HTTP_PROXY`) to match our internal convention?